docs/source/en/task_summary.mdx

```diff
@@ -10,1125 +10,224 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
-translation task, various approaches are described in this [document](https://github.com/huggingface/transformers/tree/main/examples/pytorch/translation/README.md).
+In any NLP task, text is preprocessed by separating the sequence of text into individual words or subwords. These are known as [tokens](/glossary#token). Token classification assigns each token a label from a predefined set of classes. Two common types of token classification are:
 
-Here is an example of using the pipelines to do translation. It leverages a T5 model that was only pre-trained on a
-multi-task mixture dataset (including WMT), yet, yielding impressive translation results.
+* named entity recognition (NER): label a token according to an entity category like organization, person, location or date. NER is especially popular in the biomedical domain where it can be used to label genes, proteins, and drug names.
+* part-of-speech tagging (POS): label a token according to its part-of-speech like noun, verb, or adjective. POS is useful for helping translation systems understand how two identical words are grammatically different (bank as a noun versus bank as a verb).
 
 ```py
 >>> from transformers import pipeline
 
->>> translator = pipeline("translation_en_to_de")
->>> print(translator("Hugging Face is a technology company based in New York and Paris", max_length=40))
-[{'translation_text': 'Hugging Face ist ein Technologieunternehmen mit Sitz in New York und Paris.'}]
+>>> classifier = pipeline(task="ner")
+>>> classifier("Hugging Face is a French company based in New York City.")
 ```
 
-Because the translation pipeline depends on the `PreTrainedModel.generate()` method, we can override the default
-arguments of `PreTrainedModel.generate()` directly in the pipeline as is shown for `max_length` above.
-
-Here is an example of doing translation using a model and a tokenizer. The process is the following:
+### Question answering
 
-1. Instantiate a tokenizer and a model from the checkpoint name. Summarization is usually done using an encoder-decoder
-   model, such as `Bart` or `T5`.
-2. Define the article that should be summarized.
-3. Add the T5 specific prefix "translate English to German: "
-4. Use the `PreTrainedModel.generate()` method to perform the translation.
+Question answering is another token-level task that returns an answer to a question, sometimes with context (open-domain) and other times without any context (closed-domain). There are two common types of question answering:
 
-<frameworkcontent>
-<pt>
-```py
->>> from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
-
->>> model = AutoModelForSeq2SeqLM.from_pretrained("t5-base")
->>> tokenizer = AutoTokenizer.from_pretrained("t5-base")
+* extractive: given a question and some context, the answer is a span of text from the context the model must extract
+* abstractive: given a question and some context, the answer is generated from the context
```

Comment:
This one is not dealt with by the question-answering pipeline but by the seq2seq pipeline. This should be mentioned to the user.